### PR TITLE
Introduce release template

### PR DESCRIPTION
**Pull request by Goooler**
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - title: 🐛 Bugfixes
+      labels:
+        - bug-fix
+    - title: ✏️ Tasks
+      labels:
+        - '*'
+      exclude:
+        labels:
+          - feature
+          - bug-fix
```
Comment (Goooler) [inline .github/release.yml:17]: Added labels for these, https://github.com/ComposeGears/Valkyrie/labels

Comment (Goooler) [reply .github/release.yml:17]: Labeled https://github.com/ComposeGears/Valkyrie/releases/tag/Nightly to https://github.com/ComposeGears/Valkyrie/pulls?q=is:pr+label:feature+is:closed

Comment (egorikftp) [reply .github/release.yml:17]: Nice feature! 
Do I understand correctly that all tasks should be covered by the 'housekeeping' label?"

Comment (Goooler) [reply .github/release.yml:17]: `✏️ Tasks` now contains all but `feature` and `bug-fix` (`housekeeping`, `dependencies` and other categories).

Comment (egorikftp) [reply .github/release.yml:17]: Ok, let's process with this, I like automation 🌚

**Review by egorikftp — APPROVED** [review]

File: .github/release.yml (hunk 1 above)
Comment (Goooler) [reply .github/release.yml:17]: Or we can migrate to using [keep-a-changelog](https://github.com/olivierlacan/keep-a-changelog) and auto-release like https://github.com/olivierlacan/keep-a-changelog/pull/629.

Comment (egorikftp) [reply .github/release.yml:17]: Changelog generation into file can be useful for integration with this plugin feature: https://github.com/JetBrains/gradle-changelog-plugin

It allows to see changelog in Jetbrains marketplace

Comment (Goooler) [reply .github/release.yml:17]: Yep.

**Comment by Goooler** [comment]
It's https://github.com/ComposeGears/Valkyrie/releases/tag/Nightly

File: .github/release.yml (hunk 1 above)
Comment (egorikftp) [reply .github/release.yml:17]: @Goooler 
Checked generated report, looks pretty nice, thanks for contribution ❤️ 

I see few things to improve:
- can we add "🆙 Dependencies" section based on label (for easy track changes)?
- is it possible to change title from "Nightly" to smth like "0.0.6-Nightly"?

image

Comment (Goooler) [reply .github/release.yml:17]: 1. Yes we can.
2. We can keep the version name like 0.0.6-SNAPSHOT.

Comment (egorikftp) [reply .github/release.yml:17]: yes, perfect! 
This is not urgent at all 🙂

**Pull request merged**